mlir/include/mlir/Dialect/Rock/Tuning/GridwiseGemmParams.h

```diff
@@ -41,14 +41,16 @@ struct InitParams {
   int64_t gemmMPerBlock;
   int64_t gemmNPerBlock;
   int64_t gemmKPerBlock;
+  int64_t gemmKPack;
```

Comment:
@krzysz00 I think kPack wasn't a tuning parameter in nonxdlops kernel? So this might have to be reverted.

Reply:
Yeah, kPack isn't a tuning parameter in non-xdlops, and we can't just add it like this.